config/locales/en/loans.en.yml

```diff
@@ -69,6 +69,13 @@ en:
       frozen: "Frozen"
       liquidated: "Liquidated"
     end_date: "End Date"
+    tab:
+      details: "Details"
+      questions: "Questions"
+      timeline: "Timeline"
+      timeline_list: "Timeline - List"
+      logs: "Logs"
+      calendar: "Calendar"
```

Comment:
Nice that you've consolidated these but can you please remove the originals if they're no longer needed?

So, several instances of old stuff not getting cleaned out here. I bet at some point you thought to yourself "I need to clean this out later" but forgot? If so, do you use a checklist while you're developing? I use Evernote for this b/c it has nice checkboxes:

![image](https://cloud.githubusercontent.com/assets/702320/23368742/d66dd0d8-fcdc-11e6-98ba-b74b76123950.png)

So just a tip in case it's useful.